pkg/internal/computeresources/limitrange/limitrange_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	fakekubeclient "knative.dev/pkg/client/injection/kube/client/fake"
 	fakelimitrangeinformer "knative.dev/pkg/client/injection/kube/informers/core/v1/limitrange/fake"
 	fakeserviceaccountinformer "knative.dev/pkg/client/injection/kube/informers/core/v1/serviceaccount/fake"
+	_ "knative.dev/pkg/system/testing"
```

Comment:
import side effects are pretty dangerous, and 'system testing' import side effects seem especially mysterious - if we can't remove this now and we have to include it to make the unit tests pass, can you create an issue to dig into what is going on here and see if we can avoid this side effect import?

+ @jerop

Reply:
There are some other places in our code base using this. It may be worth to look into it why and when we need this import. 
https://github.com/tektoncd/pipeline/search?q=_+%22knative.dev%2Fpkg%2Fsystem%2Ftesting%22